src/setup/index.js

```diff
@@ -22,7 +22,6 @@ export default function () {
     Onyx.init({
         keys: ONYXKEYS,
         safeEvictionKeys: [ONYXKEYS.COLLECTION.REPORT_ACTIONS],
-        keysToDisableSyncEvents: [ONYXKEYS.CURRENTLY_VIEWED_REPORTID],
```

Comment:
Probably we don't need this feature in Onyx. I think we added it so that the reportID would stay "synced" but if we are using a local value instead of Onyx then we won't have this problem.

Reply:
Great! Let's remove it